src/work-allocation-2/enums/priority-limits.ts

```diff
@@ -0,0 +1,4 @@
+export enum PriorityLimits {
+    Urgent= 2000,
+    High= 5000,
```

Comment:
Minor comment - spacing of equals is off here (see below enum)

Reply:
Done